app/code/Magento/Eav/Model/Entity/Attribute/OptionManagement.php

```diff
@@ -62,14 +62,20 @@ public function add($entityType, $attributeCode, $option)
 
         $label = trim((string)$option->getLabel());
         if ($label === '') {
-            throw new InputException(__('The attribute option label is empty. Enter the value and try again.'));
+            throw new InputException(
+                __(
+                    'The attribute option label for attribute \'%1\' is empty. Enter the value and try again.',
+                    $attribute->getAttributeCode()
+                )
+            );
         }
 
         if ($attribute->getSource()->getOptionId($label) !== null) {
             throw new InputException(
                 __(
-                    'Admin store attribute option label "%1" is already exists.',
-                    $option->getLabel()
+                    'Admin store attribute option label "%1" for attribute \'%2\' already exists.',
+                    $option->getLabel(),
+                    $$attribute->getAttributeCode()
```

Comment:
I think it is a typo here:
```suggestion
                    $attribute->getAttributeCode()
```